test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -359,11 +359,10 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 			},
 			Prebind: nil,
 		}
+		noClass := ""
 		pvcConfig := framework.PersistentVolumeClaimConfig{
-			Annotations: map[string]string{
```

Comment:
FYI we will need to backport all the e2e test fixes to 1.10 as well.  Upgrade tests will run 1.10 versions of e2e tests against a 1.11 cluster.